subjects/java/checkpoints/almost-palindrom/README.md

```diff
@@ -0,0 +1,39 @@
+## Palindrome
+
+### Instructions
+
+In a file named `AlmostPalindrome.java` write a function `isAlmostPalindrome` that returns true if the String in parameter is almost palindrome.
+A word is considered "almost a palindrome" if removing one letter from the word makes it a palindrome.
```

Comment:
What should happen for edge cases? Is an empty string "almost palindrome"?

Reply:
They will get just string formed with at least 3 letters, i will add that.